database/migrations/2021_12_11_134404_alter_coaches_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class AlterCoachesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+    Schema::table('coaches', function (Blueprint $table) {
+        $table->string('address', 225)->change();
+        
+    });
+    Schema::table('coaches', function (Blueprint $table) {
```

Comment:
why not in single table statement?

Reply:
Column adding is missing

Reply:
fixed

Reply:
still no alter table approach, that adds new column.